ReverseTicTacToe/UI/ConsoleUIManager.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System;
+using System.Text;
+using static System.Console;
+using ReverseTicTacToe.Logic;
+using Ex02.ConsoleUtils;    
+
+namespace ReverseTicTacToe.UI
+{
+    class ConsoleUIManager
+    {
+        private const int k_MinBoardSize = 3;
+        private const int k_MaxBoardSize = 9;
+        private const int k_MinGameType = 1;
+        private const int k_MaxGameType = 2;
+        private const int k_NumOfDimensions = 2;
+        private const int k_InitVal = -1;
+        private const int k_MinBoardIndex = 0;
+        private const int k_RowIndex = 0;
+        private const int k_ColIndex = 1;
+
+        private const string k_QuitSymbol = "Q";
+        private string errorMessage = "";
```

Comment:
are you sure it's ok to initialize a non constant non static member here? and not in the constructor?

Reply:
fixed. moved init to inside function